1.cellprofiler_ic/nf1_ic.sh

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 
-# activate the main conda environment (if not already activated)
+# this line is needed for the sh file to properly activate a conda environment 
 eval "$(conda shell.bash hook)"
```

Comment:
`eval` is considered a security issue. Ask @d33bs more about this as he was the one who brought this to my attention initially.

Reply:
@d33bs is reviewing now! :smile:

Reply:
I have made a change (thanks to ChatGPT) to not use `eval` like I found from StackOverflow. I have changed to `conda init <SHELL_NAME>' instead as this works but the only issue is that the user must change the shell name to their local machine shell (e.g. Linux = `bash`, MacOS = `zsh`, etc.). I have added that note into the README but good thing is that this avoids the `eval` function.